recipes/mlrose-ky/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mlrose-ky" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/d9/12/db9f576553caa2df5b435a8e941ff29426c199e5aaee28b6967a97d4682a/mlrose_ky-1.1.6.tar.gz
```

Comment:
Please use the canonical url so that the update bot can generate automatic PRs:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

Reply:
`https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mlrose_ky-{{ version }}.tar.gz`

Reply:
Done. [2dc7527](https://github.com/conda-forge/staged-recipes/pull/27859/commits/2dc7527f0fc426d4720e09141da65cf8950220c3)